index.html

```diff
@@ -305,18 +305,24 @@ <h6 class="uppercase mb16">Technologies used: Kotlin, Java, Shell Script, Python
 </section>
 <section>
     <div class="container">
-        <div class="row">
+        <div class="row text-center">
             <div class="col-sm-4">
                 <h2 class="number">Phimp.me</h2>
                 <p>
                     Do you have many different apps in your Android Phone for the purpose of clicking, editing and sharing images? Hey guess what, now you don't need all of those because Phimp.me is here, with all the features packaged into one. So go and clean your phone memory to save your space for other useful purposes.
-                </p>
-            </div>
+                </p>                
+                <div class="row" style="margin-top:75px">
```

Comment:
Please find another alternative to `margin-top`. you can define size of container and you can position button relative to container